cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageTable.java

```diff
@@ -91,7 +92,7 @@ public static FieldLineageTable create(StructuredTableContext structuredTableCon
     return new FieldLineageTable(structuredTableContext);
   }
 
-  private StructuredTable getEndpointChecksumTable() {
+  public StructuredTable getEndpointChecksumTable() {
```

Comment:
is there a reason this needs to be public now?

Reply:
Oops this seems to be by mistake